packages/app/src/cli/services/dev/output.ts

```diff
@@ -76,8 +76,7 @@ function outputFunctionsMessage(extensions: ExtensionInstance[]) {
   if (extensions.length === 0) return
   const names = extensions.map((ext) => ext.configuration.name)
   const heading = outputToken.heading(names.join(', '))
-  const message = `These extensions need to be deployed to be manually tested.
-One testing option is to use a separate app dedicated to staging.`
+  const message = `Your app must be installed to test these extensions. Use the Preview URL to open your App home.`
```

Comment:
@isaacroldan @MeredithCastile What do you think of this new message? The app no longer needs to be deployed, but it will still need to be installed for the functions to be testable. The exact steps after this will depend on the function API (e.g. create a discount or enable a checkout validation)

Reply:
Looks good, this is also a requirement for ui extensions so I think it's good to show this.

Reply:
Yeah, first step always needs to be to install the app, UI Previews won't work otherwise, weird that we are only show this for functions though

Reply:
Should we leave out the functions-specific messaging entirely now then?

Reply:
Talked with @MeredithCastile, we can remove the functions-specific messages here 🎉

Reply:
✅